apps/print/frontend/src/utils.ts

```diff
@@ -1,7 +1,59 @@
+import type { BallotPrintCount } from '@votingworks/print-backend';
 import { Election, LanguageCode } from '@votingworks/types';
 
+function sortLanguages(
```

Comment:
Added this in so language order stays consistent. I noticed this because an election I was testing with had Spanish first in the election definition. It looked a bit od to have the default value of English not be the first radio button, so I added this ordering, following the order in VxDesign.